index.html

```diff
@@ -72,6 +75,74 @@ <h2>
 
                 <!-- Please keep the team pages in alphabetical order, too. -->
 
+                <!-- Best Practices Team -->
+                <section data-background-image="images/best-practices-team.webp" data-background-color="#b6ccd3">
```

Comment:
The default text colour doesn't work with every background image.
Is there a way to alter the text colour per slide?

Reply:
Yes:

```suggestion
                <section data-background-image="images/best-practices-team.webp" data-background-color="#b6ccd3" style="color: #fff;">
```

(You might need to set the color for the heading in the `h2` tag as it has its own styling.)

Reply:
I guess we actually need some blur/contrast/surrounding, etc. I'm not a designer or frontend and don't know the proper names.

The image has many colours which makes it hard to find a single suiting colour.

Reply:
You can surround the `h2` and `ul` with a `div` that has a semi-transparent background, e.g., like this:

```html
<div style="background-color:rgba(182, 204, 211, .8);">
```

Reply:
Thanks, that works out pretty well. Adjusted the PR and closing this discussion.